packages/blockchain-wallet-v4-frontend/src/data/components/swap/sagas.ts

```diff
@@ -222,6 +222,7 @@ export default ({
       }
       yield put(actions.form.stopSubmit('previewSwap'))
       yield put(actions.components.refresh.refreshClicked())
+      yield put(actions.form.destroy('swapAmount'))
```

Comment:
May or may not help: I think this is automatically done when unmounting the reduxForm component. Maybe your component has `destroyOnUnmount: false`.

Reply:
i believe it does, @milan-bc can you check that. removing that would be the correct way to fix this

Reply:
we do have it `destroyOnUnmount: false` but what I notice then if you go forward and want to return back to enter amount page we lost entered amount in that case, that is why I added destroy after we do create order

Reply:
we have preview swap where user can decide to go back, UX thingy we have to keep form value on unmount in such case